### DISCUSSION TITLE
Tap Artist or Album name in playing track to go to Artist or Album

### DISCUSSION THREAD

**Discussion by thievingjoker** [MoleMan1024/audiowagon#130] [Ideas]
Would it be possible to make it so the Artist name or Album name display was actually linked to the Artist or Album (or submits it as a search that would get there)?

I frequently listen to a playlist,, and as I am listening to a song I think I'd like to quickly go to that Artist or Album (without having to search). 

That would be super handy if you could just tap on the artist or album playing and go right there.

**Comment by MoleMan1024** [comment]
hi,
as far as I know, that is not possible given how the default MediaBrowser in Android works.
